Dockerfile

```diff
@@ -61,19 +61,32 @@ RUN apt-get update -qq && \
     apt-get install --no-install-recommends -y curl libpq-dev libvips libjemalloc2 && \
     apt-get clean
 
+ENV USERNAME rails_api_base
+ENV USER_UID 1000
+ENV USER_GID 1000
```

Comment:
Should we use `ENV` or `ARG` here? 🤔 https://docs.docker.com/build/building/variables/

Reply:
At first I tried using ARG but I remember I had some issues with that. I will take another look.